analytics/index.yaml

```diff
@@ -101,11 +101,10 @@ paths:
           example: http://example.com/path/to/previous/page.html
         - name: _cvar
           in: query
-          #TODO: I don't understand this parameter or it's format. HALP!
-          #TODO: Confirm serialization format (it's supposed to be Array JSON, but example suggests Array-like Object JSON).
           #TODO: Confirm description of item format.
           #TODO: Add limits of item values (e.g. max length).
           #TODO: Link to a page describing custom variables.
+          #TODO: Link to a definition of scopes.
           description: |
             <a href="https://help.piwik.pro/analytics/custom-variables/">Custom variables</a> list set on the ``visit``
             scope.
```

Comment:
Description should explain format used by this parameter. Current one doesn't explain anything and reader has to guess it from example.

Reply:
The description can be filled on a basis of Matomo docs (custom variables in PPMS Anlytics are 100% compatible: https://developer.matomo.org/api-reference/tracking-api#optional-user-info